cli/server.go

```diff
@@ -254,6 +254,7 @@ func server() *cobra.Command {
 				Logger:               logger.Named("coderd"),
 				Database:             databasefake.New(),
 				Pubsub:               database.NewPubsubInMemory(),
+				CacheDir:             cacheDir,
```

Comment:
`cacheDir` defaults to `/tmp/` on my system. We should probably change that to not use `/tmp` at all and instead use `~/.cache` so we're not dumping 300MB into RAM on startup

Reply:
Indeed, I've hinted at this in #2200 but we should create a separate issue for it.

Reply:
https://github.com/coder/coder/issues/2534